llmware/library.py

```diff
@@ -383,7 +486,18 @@ def get_knowledge_graph_status (self):
 
     def set_knowledge_graph_status (self, status_message):
 
-        """ Updates the knowledge graph status on the library card after creating a knowledge graph """
+        """ Updates the knowledge graph status on the library card after creating a knowledge graph 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Updates the knowledge graph status on the library card after creating a knowledge graph.
```